cleanlab/multilabel_classification/filter.py

```diff
@@ -0,0 +1,133 @@
+import warnings
```

Comment:
TODO: add another method here: `find_label_issues` which is just a wrapper around the current: `_find_label_issues_multilabel()`

But with user-friendly documentation

Reply:
Added here: https://github.com/cleanlab/cleanlab/blob/361e05284c5bb5151a49aee7e5aa8ff41ad47220/cleanlab/multilabel_classification/filter.py#L6